external-libraries.txt

```diff
@@ -80,6 +80,11 @@ Project: Apache Commons CLI
 URL:     http://commons.apache.org/cli/
 License: Apache-2.0
 
+Id:      commons-io:commons-io
```

Comment:
Thank you for updating this file! - let's see if we really need to include these libraries.